### PR TITLE
Update react-native.mdx

### PR DESCRIPTION
**Pull request by ahwreck**
Updating react-native-radar documentation to include commonly overlooked steps on iOS and Android.

### CODE REVIEW

**Comment by matthewisabel** [comment]
@andlum do you mind taking a look at this with the other react native changes? I imagine we want this to reuse as much as possible the [iOS](https://radar.com/documentation/sdk/ios#request-permissions) / [Android](https://radar.com/documentation/sdk/ios#request-permissions) request permissions sections.

**Review by andlum — APPROVED** [review]
This is great! Thank you for adding this to the RN Docs.

File: docs/sdk/react-native.mdx [hunk 1]
```diff
@@ -28,7 +28,23 @@ Then, install the [iOS SDK](/sdk/ios). Change to the `ios/` directory. In the `P
...
+### iOS
+
+You must add location usage descriptions and background modes to your `Info.plist`. For foreground permissions, you must add a value for  `NSLocationWhenInUseUsageDescription` *(Privacy - Location When In Use Usage Description)*. For background permissions, you must add a value for `NSLocationAlwaysAndWhenInUseUsageDescription` *(Privacy - Location Always and When In Use Usage Description)*. These strings are displayed in permissions prompts. 
+
+<img
+  className="large"
+  src="https://images.ctfassets.net/f2vbu16fzuly/66LUD7W9uTt7f2H5pMmpBp/1e5a2bb2baaf79328e88c3ea47a7f6ff/permissions.png"
+/>
+
+If you are requesting background permissions, in your project settings, go to _Signing & Capabilities_, add _Background Modes_, and turn on _Background fetch_ and _Location updates_. Note that this requires additional justification during App Store review. Learn more [below](#submit-to-app-store).
```
Comment (brettguenther) [inline docs/sdk/react-native.mdx:40]: should we scope location to just using tracking options that include time interval based updates (includes responsive and continuous and non 0 values for moving or stopped update intervals)?

Comment (brettguenther) [inline docs/sdk/react-native.mdx:40]: don't think submit to app store exists on this page. can link to ios sdk page or copy here.

Comment (jsani-radar) [reply docs/sdk/react-native.mdx:40]: slightly confused - what outcome are you trying to drive toward here?

**Pull request closed without merging**